README.md

```diff
@@ -66,8 +66,10 @@ export default {
 - [x] Support first release (no previous tags)
 - [x] Support defining next version manually (can be done by changing the version on the release pull-request title / commit message)
 - [ ] Support more forges:
-  - [ ] Gitea
+  - [x] Github
+  - [x] Gitea
```

Comment:
```suggestion
  - [x] Gitea / Forgejo
```

Reply:
I feel like Forgejo should be treated as its own thing, given it hard-forked and could get breaking changes at any point.

Reply:
@Andre601 well if they have there own sdk ... 

they currently dont: https://www.npmjs.com/search?q=forgejo

Reply:
@6543 A separate SDK for Forgejo is already in use: https://github.com/woodpecker-ci/woodpecker/blob/b1cf44ae0f80ca18dff3ab67b61cda59107b5a39/go.mod#L12

Reply:
As we already use a separate SDK in WP and the hard-fork has been announced, I also think that treating them individually makes more sense.

Reply:
well I updated the readme ... if there is an acutall forgejo sdk that is not ages old and maintained ... we can switch to it at any time ...